src/compiler/utilities.ts

```diff
@@ -693,7 +693,7 @@ namespace ts {
                 // At this point, node is either a qualified name or an identifier
                 Debug.assert(node.kind === SyntaxKind.Identifier || node.kind === SyntaxKind.QualifiedName || node.kind === SyntaxKind.PropertyAccessExpression,
                     "'node' was expected to be a qualified name, identifier or property access in 'isPartOfTypeNode'.");
-                // falls through
+            // falls through

```

Comment:
Below too.

---
In reply to: [132297135](https://github.com/Microsoft/TypeScript/pull/17625#discussion_r132297135) [](ancestors = 132297135)